gradle/libs.versions.toml

```diff
@@ -2,24 +2,19 @@
 
 detektGradlePlugin = "1.23.6"
 junit = "4.13.2"
-kotlinExposed = "1.4.32"
+kotlinExposed = "1.7.21"
 dokka = "1.9.20"
 lint = "30.1.0"
 openTelemetryCore = "1.38.0"
 openTelementrySemConv = "1.25.0-alpha"
-moshi = "1.12.0"
-lifecycle = "2.5.0"
-lifecycle-extensions = "2.0.0"
-mockk = "1.13.11"
-androidx-test = "1.6.1"
-androidx-test-junit = "1.2.1"
-robolectric = "4.13"
-mockwebserver = "4.12.0"
+moshi = "1.14.0"
```

Comment:
This change doesn't move the Moshi codegen to use KSP. That will come in a different PR.